pylmod/gradebook.py

```diff
@@ -915,8 +942,50 @@ def _spreadsheet2gradebook_multi(
                         name = field
                         shortname = field[0:3] + field[-2:]
                         log.info('calling create_assignment from multi')
+
+                        # If the max_pts and normalize columns are present,
+                        # and use_max_points_column is True,
+                        # replace the default value for max points.
+                        max_points = 1.0  # the default
```

Comment:
This is a python library, there is no settings.py. I'll move this into a constant at the top of the file